src/stores/orderStore.ts

```diff
@@ -208,10 +191,17 @@ class OrderStore extends BaseStore {
           Types.Expired
         );
       } else {
-        this.notificationStore.addNotification(
-          levels.error,
-          `${error.message}`
-        );
+        if (error.status === 500) {
```

Comment:
Create a variable for errorMessage, fill it using this condition and then put it into notifications list